pkg/dependencies/version.go

```diff
@@ -0,0 +1,70 @@
+package dependencies
+
+import (
+	"fmt"
+	"os/exec"
+	"regexp"
+
+	"github.com/Masterminds/semver"
+	"github.com/pkg/errors"
+)
+
+// returns the currently installed clang version
+func clangVersion(dep *Dependency) (*semver.Version, error) {
+	path, err := dep.finder.ClangPath()
+	if err != nil {
+		return nil, err
+	}
+
+	re := regexp.MustCompile(`(?m)^clang version (?P<version>\d+.\d+.\d+)$`)
```

Comment:
On my system, the output is `Ubuntu clang version 14.0.0-1ubuntu1`. How about:
```suggestion
	re := regexp.MustCompile(`(?m)clang version (?P<version>\d+.\d+.\d+)$`)
```